sherpa/astro/xspec/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright (C) 2010, 2015 - 2024
+#  Copyright (C) 2010, 2015, 2024
```

Comment:
Unfortunately your "ensure a sensible copyright line" script doesn't handle ranges, so it's messed it up here (and in at least one other flle).

Reply:
I wosh we could just drop the years form the copyright line. Would be way easier/ Many projects don't do it at all and copyright last so many decades that a year more or less has no practical relevance. Anyway, not for me to decide. It is what it is...